backend/main/leaderboard_test.go

```diff
@@ -34,4 +36,33 @@ func TestGetCommunityLeaderboard(t *testing.T) {
 	assert.Equal(t, 2, len(p.Data))
 	assert.Equal(t, 3, p.Data[0].Score)
 	assert.Equal(t, 2, p.Data[1].Score)
+}
+
+func TestGetCommunityLeaderboardWithEarlyVotes(t *testing.T) {
+	clearTable("communities")
+	clearTable("community_users")
+	clearTable("proposals")
+	clearTable("votes")
```

Comment:
should the `community_users_achievements` be cleared before this test runs?

Reply:
Good catch.